File: HK3/CongNgheLapTrinhHienDai/TaiLieu_BaiTap/D05/1642021_NodeResful/1642021_Api/models/taiKhoanModel.js
var db = require('../fn_dao/dbconnect_mysql');

exports.loadAllTK = () => {
    var sql = 'select * from the';
    var obj = {};
    // db.load(sql).then(rows => {
    //     return rows;
    // });
    console.log(db.load(sql));
    return db.load(sql);
};

exports.loadTaiKhoan = (mathe, matkhau, nganhang) => {
    let sql = `select t.MaThe MaThe,
                        t.TenChuThe TenChuThe,
                        t.SoDuKhaDung SoDuKhaDung,
                        t.NgayHetHan NgayHetHan,
                        t.MatKhau MatKhau,
                        nh.TenNganHang TenNganHang,
                        nh.MaNganHang MaNganHang
                from the t,nganhang nh
                where t.NganHang = nh.MaNganHang 
                                and t.MaThe = ${mathe}
                                and t.MatKhau = ${matkhau}
                                and t.NganHang = ${nganhang}`;
    return db.load(sql);
};

exports.loadThongTinTaiKhoan = (mathe, nganhang) => {
    let sql = `select t.MaThe MaThe,
                    t.TenChuThe TenChuThe,
                    t.SoDuKhaDung SoDuKhaDung,
                    t.NgayHetHan NgayHetHan,
                    nh.TenNganHang TenNganHang,
                    nh.MaNganHang MaNganHang
                from the t,nganhang nh
                where t.NganHang = nh.MaNganHang 
                            and t.MaThe = ${mathe}
                            and t.NganHang = ${nganhang}`;
    return db.load(sql);
};

File: HK3/CongNgheLapTrinhHienDai/TaiLieu_BaiTap/D05/1642021/1642021ApiClient/Model/TaiKhoanModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _1642021ApiClient.Model
{
    public class TaiKhoanModel
    {
        public int MaThe { get; set; }
        public string TenChuThe { get; set; }
        public System.DateTime NgayHetHan { get; set; }
        public double SoDuKhaDung { get; set; }
        public int NganHang { get; set; }
        public string MatKhau { get; set; }
        public string TenNganHang { get; set; }
        public TaiKhoanModel()
        {

        }
        public TaiKhoanModel(TaiKhoanModel the)
        {
            MaThe = the.MaThe;
            MatKhau = the.MatKhau;
            TenChuThe = the.TenChuThe;
            NgayHetHan = the.NgayHetHan;
            SoDuKhaDung = the.SoDuKhaDung;
            NganHang = (int)the.NganHang;
        }
    }
}
